rpm/user-managerd.spec

```diff
@@ -51,6 +55,11 @@ make %{?_smp_mflags}
 %install
 make -C build INSTALL_ROOT=%{buildroot} install
 mkdir -p %{buildroot}%{_datadir}/user-managerd/remove.d
+mkdir -p %{buildroot}%{_unitdir}/user@105000.service.wants/
+ln -s ../tmp-guest_home.mount %{buildroot}%{_unitdir}/user@105000.service.wants/
+mkdir -p %{buildroot}%{_unitdir}/autologin@105000.service.wants/
+ln -s ../tmp-guest_home.mount %{buildroot}%{_unitdir}/autologin@105000.service.wants/
+ln -s ../guest_disable_suw.service %{buildroot}%{_unitdir}/autologin@105000.service.wants/
```

Comment:
Failing to see / find guest_disable_suw.service file. Maybe you forgot to git add it?

Reply:
Yes, that's what happened. I rewrote the commits and forgot to call add.